lib/screen/irl_screen/webview_irl_screen.dart

```diff
@@ -87,30 +92,40 @@ class _IRLWebScreenState extends State<IRLWebScreen> {
           JSResult.error('Blockchain is unsupported'),
         );
       }
-      final addresses = await injector<CloudDatabase>()
-          .addressDao
-          .getAddressesByType(cryptoType.source);
+      final addresses = await _getWalletAddress(cryptoType);
       if (addresses.isEmpty) {
-        final persona =
-            await injector<AccountService>().getOrCreateDefaultPersona();
-        final addedAddress = await persona.insertNextAddress(
+        final personas =
```

Comment:
please use account service

Reply:
account service does not support that currently, beside, it has to query persona again